### PR TITLE
refactor maxattempt so that they are configurable individually

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

**Review by damianxu88 — APPROVED** [review]
LGTM

File: spade-aws/src/main/scala/com/salesforce/mce/spade/aws/activity/EmrActivity.scala [hunk 1]
```diff
@@ -24,13 +25,15 @@ object EmrActivity {
   case class Builder(
     nameOpt: Option[String],
     steps: Seq[EmrActivitySpec.Step],
-    runsOn: Resource[EmrCluster]
+    runsOn: Resource[EmrCluster],
+    maxAttempty: Option[Int]
```
Comment (schowsf) [inline spade-aws/src/main/scala/com/salesforce/mce/spade/aws/activity/EmrActivity.scala]: typo?  maxAttempt

Comment (realstraw) [reply spade-aws/src/main/scala/com/salesforce/mce/spade/aws/activity/EmrActivity.scala]: good catch!

**Review by schowsf — APPROVED** [review]
LGTM

**Review by damianxu88 — APPROVED** [review]
LGTM

**Pull request merged**